scrapy/cmdline.py

```diff
@@ -18,10 +18,10 @@ def _iter_command_classes(module_name):
     # TODO: add `name` attribute to commands and and merge this function with
     # scrapy.utils.spider.iter_spider_classes
     for module in walk_modules(module_name):
-        for obj in vars(module).itervalues():
+        for _, obj in vars(module).items():
```

Comment:
Certainly. I blindly copied from the py3 porting guide which showed an items() example, and then I completely forgot values() ;)